pkg/microservice/aslan/core/common/service/registry/service_test.go

```diff
@@ -137,3 +140,33 @@ func TestReverseStringSlice_Len(t *testing.T) {
 		t.Error("reverse sort not works")
 	}
 }
+
+func TestSwrListRepoImage(t *testing.T) {
+	s := &SwrService{}
+	listRepoImagesOption := ListRepoImagesOption{
+		Endpoint: Endpoint{
+			Namespace: "lilian",
+			Ak:        "",
+			Sk:        "",
+		},
+		Repos: []string{"nginx-test"},
+	}
+	_, err := s.ListRepoImages(listRepoImagesOption, log.SugaredLogger())
+	assert.Nil(t, err)
+}
+
+func TestSwrImageInfo(t *testing.T) {
+	s := &SwrService{}
+	getRepoImageDetailOption := GetRepoImageDetailOption{
+		Endpoint: Endpoint{
+			Region:    "cn-north-4",
+			Namespace: "lilian",
+			Ak:        "",
+			Sk:        "",
+		},
+		Tag:   "20210712210942-34-master",
+		Image: "nginx-test",
+	}
+	_, err := s.GetImageInfo(getRepoImageDetailOption, log.SugaredLogger())
+	assert.Nil(t, err)
+}
```

Comment:
It's good to see that you added some unittests. But these two cases are meaningless. To test these two methods, you need first mock the swr client, and check if the return value is as expected, not just check the error.

Reply:
error is not equal to empty is in line with my expectations